libraries/resource_hollandbackup_backupset.rb

```diff
@@ -0,0 +1,68 @@
+require 'chef/resource/lwrp_base'
+
+class Chef
+  class Resource
+    class HollandbackupBackupset < Chef::Resource::LWRPBase
+      self.resource_name = :hollandbackup_backupset
+      actions :configure
+      default_action :configure
+
+      # template
+      attribute :name, :kind_of => String, :name_attribute => true
```

Comment:
Some of these appear to be duplicated across different resources. Consider a shared parent class, or use poise's helper for template content - https://github.com/poise/poise#template-content.